Objects/obmalloc.c

```diff
@@ -2612,6 +2615,37 @@ _PyObject_DebugDumpAddress(const void *p)
     _PyMem_DumpTraceback(fileno(stderr), p);
 }
 
+static void
+free_arenas(PyInterpreterState *interp)
+{
+    OMState *state = &interp->obmalloc;
+    for (uint i = 0; i < maxarenas; ++i) {
+        // free each obmalloc memory arena
+        struct arena_object *ao = &allarenas[i];
+        _PyObject_Arena.free(_PyObject_Arena.ctx,
+                             (void *)ao->address, ARENA_SIZE);
+    }
+    // free the array containing pointers to all arenas
+    PyMem_RawFree(allarenas);
+#if WITH_PYMALLOC_RADIX_TREE
```

Comment:
Current `_freeze_module` is a 32-bit application on Windows, which does not define the `USE_INTERIOR_NODES` flag during the build.